WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -53,7 +53,7 @@ public static bool TryParseError(JToken token, [NotNullWhen(true)] out HwiExcept
 		}
 
 		var errToken = token["error"];
-		var codeToken = token["code"];
+		JToken? codeToken = token["code"];
```

Comment:
No need for this change.

Reply:
Or alternatively convert also L55 to an explicit type then it would be clear that `token[something]` can be nullable (not obvious)